lib/supabase/mock.ts

```
@@ -0,0 +1,358 @@
+            }
+
+            if (callback) {
+              callback({ data, error: null })
+              return { data, error: null }
+            }
+            return { data, error: null }
+          }
+        })
+      }),
+
+      then: async (callback: any) => {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)